salt/modules/dockerng.py

```diff
@@ -5529,3 +5540,216 @@ def script_retcode(name,
                    ignore_retcode=ignore_retcode,
                    use_vt=use_vt,
                    keep_env=keep_env)['retcode']
+
+
+def _mk_fileclient():
+    '''
+    Create a file client and add it to the context.
+    '''
+    if 'cp.fileclient' not in __context__:
+        __context__['cp.fileclient'] = \
+                salt.fileclient.get_file_client(__opts__)
```

Comment:
Fixed.
As I showed @isbm, main problem is that running ` pylint --rcfile=.testing.pylintrc --disable=W1307 salt/modules/dockerng` does not have the same behavior as what I see in Jenkins.